doc/userguide/configuration/exception-policies.rst

```diff
@@ -327,6 +325,13 @@ The available command-line options are:
 - ``simulate-alert-queue-realloc-failure``: prevent the engine from dynamically
   growing the temporary alert queue, during alerts processing.
 
+Glossary
+========
+
+- **inspection**: traffic is inspected for events, anomalies and logging;
```

Comment:
this is not a distinction we many anywhere I think

Reply:
Is it a requested change for this PR ?

Reply:
change inspection to parsing.